pallets/bridge/merkle-proof/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
```

Comment:
We don't have this module in any public repo yet (except for the original `snowbridge`). 
I can either reference their repo with a specific commit or publish this pallet in a separate repo, but I guess this will only complicate things.

Reply:
I was talking about the public snowbridge repo. Whats the complexity in referencing it with a commit?

Reply:
I guess it's better to minimize dependencies to some uncontrolled repo, as it can be moved, history may be overridden, and the repository can be hidden - any such move will break the build.
But anyway I will move this pallet to dependencies to reduce the amount of duplicate code.